lib/dialects/mssql/query-interface.js

```diff
@@ -1,53 +1,55 @@
 'use strict';
 
-/**
- Returns an object that treats MSSQL's inabilities to do certain queries.
-
- @class QueryInterface
- @static
- @private
- */
+const { QueryInterface } = require('../abstract/query-interface');
 
 /**
-  A wrapper that fixes MSSQL's inability to cleanly remove columns from existing tables if they have a default constraint.
-
-
-  @param  {QueryInterface} qi
-  @param  {string} tableName     The name of the table.
-  @param  {string} attributeName The name of the attribute that we want to remove.
-  @param  {object} options
-  @param  {boolean|Function} [options.logging] A function that logs the sql queries, or false for explicitly not logging these queries
-
-  @private
+ * Returns an object that treats MSSQL's inabilities to do certain queries.
+ *
+ * @private
```

Comment:
As previously noted in other comment, remove `@private` directive. Comment can be improved to indicate this is a MSSQL Query Interface